modules/server/src/main/ts/BedrockAuto.ts

```diff
@@ -68,12 +95,18 @@ export const go = (bedrockAutoSettings: BedrockAutoSettings): void => {
         return Reporter.writePollExit(settings, data);
       });
 
+      // Combine all shutdowns into single function call.)
+      const shutdown = () => Promise.allSettled(shutdownServices.map((shutdown_fn) => shutdown_fn()));
```

Comment:
Could this be defined once seems to be the same function. Also is the comment overkill, it is kind of obvious what the function does.

Reply:
Removed duplication